sdk/eventhub/event-hubs/samples/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "azure-event-hubs-samples",
+  "version": "1.0.0",
+  "description": "The samples in this folder are for version 5.0.0 and above of this library. If you are using version 2.1.0 or lower, then please use [samples for v2.1.0](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/event-hubs_2.1.0/sdk/eventhub/event-hubs/samples) instead",
+  "main": "dist/runsamples.js",
```

```suggestion
  "main": "dist-esm/runsamples.js",
```
Should this be in dist-esm? The tsconfig.json has the outDir as the dist-esm directory.

---

Yes. Although it doesn't get used since I'm just using ts-node.